### DISCUSSION TITLE
[@types/tizen-tv-webapis] Missing `player_id` property on AVPlayManager

### DISCUSSION THREAD

**Discussion by WillsonHaw** [DefinitelyTyped/DefinitelyTyped#60645] [Issues with a @types package]
When creating an `AVPlayManager` using `webapis.avplayStore.getPlayer()`, the returned `AVPlayManager` should have a `player_id: string` property. This property should not be available from the `webapis.avplay` version of AVPlayManager.

[a comment by an automated account is not shown]

**Comment by pwsses** [comment]
Hello.
You are right. 
webapis.avplayStore.getPlayer() returned AVPlayManager has a player_id property.
But it is not specified in the widl document. (https://developer.samsung.com/smarttv/develop/api-references/samsung-product-api-references/avplaystore-api.html)
I will check about what to do with this.
Thank you.
